SPECS/python-sphinx_rtd_theme/python-sphinx_rtd_theme.spec

```diff
@@ -0,0 +1,210 @@
+%global srcname sphinx_rtd_theme
+Summary:        Sphinx theme for readthedocs.org
+Name:           python-%{srcname}
+Version:        0.4.3
+Release:        12%{?dist}
+License:        MIT AND OFL
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/rtfd/%{srcname}
+Source0:        https://github.com/rtfd/%{srcname}/archive/%{version}/%{srcname}-%{version}.tar.gz#/%{name}-%{version}.tar.gz
+# The koji builders do not have network access, and this file is not included
+# in any Fedora package, so we retrieve it for offline use.
+Source1:        https://docs.readthedocs.io/en/latest/objects.inv
```

Comment:
Nit: This could probably be renamed to something including the package name for the source blobstore

Reply:
I agree, will update soon.